doc/devel/spicy/autogen/tftp-no-accept.spicy

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
```

Comment:
was this file supposed to be removed in the original PR?

Reply:
Yeah it was. It should be removed here too.